### ISSUE TITLE
fin project rsync command

### ISSUE THREAD

**Issue by lmakarov** [docksal/docksal#1524]
Implement a built-in command in fin to rsync files between host and project containers.

Example:

```
fin project rsync cli:/var/www/vendor ./vendor
fin project rsync ./vendor cli:/var/www/vendor
```

**Comment by shelane** [comment]
This might be the same as #1229

**Comment by lmakarov** [comment]
> This might be the same as #1229

True. I'll close the older one since this one is already linked in a few places.

**Comment by JDDoesDev** [comment]
@lmakarov cross reply from #1506 for reference for anyone looking at this issue: 

Here is a gist of what I have as a custom command https://gist.github.com/JDDoesDev/8049b64619530cce8469e22cd4ae837f and I am going to be including a link to it in the docs.
